ci/Jenkinsfile.tests-imports

```diff
@@ -51,7 +51,16 @@ pipeline {
     }
 
     stage('Check') {
-      steps { sh 'make run-statusq-sanity-checker' }
+      steps { script {
+        wrap([
```

Comment:
Looks good, but a comment explaining why we need this for these sanity checks would be nice.